apps/web/screens/Category/Category.tsx

```diff
@@ -268,184 +253,208 @@ const Category: Screen<CategoryProps> = ({
-                        })}
-                      </Box>
-                    </AccordionCard>
-                  </div>
-                )
-              })}
-              {lifeEvents.map(
-                ({ title, slug, intro, thumbnail, image }, index) => {
-                  return (
-                    <Card
-                      key={index}
-                      href={makePath('lifeEvent', '[slug]')}
-                      as={makePath('lifeEvent', slug)}
-                      description={intro}
-                      title={title}
-                      image={(thumbnail || image) as Image}
-                      tags={[
-                        {
-                          title: n('categoryTag'),
-                        },
-                      ]}
-                    />
-                  )
-                },
-              )}
-              {cards.map(({ title, content, slug }, index) => {
-                return (
-                  <Card
-                    key={index}
-                    title={title}
-                    description={content}
-                    href={makePath('article', '/[slug]')}
-                    as={makePath('article', slug)}
-                  />
-                )
-              })}
-            </Stack>
-          </ColorSchemeContext.Provider>
-        }
       >
         <Box paddingBottom={2}>
           <Breadcrumbs>
-            <Link href={makePath()}>Ísland.is</Link>
+            <Link href={pathNames()}>Ísland.is</Link>
           </Breadcrumbs>
         </Box>
+        <Box paddingBottom={[5, 5, 10]}>
+          <Text variant="h1" as="h1" paddingTop={[4, 4, 0]} paddingBottom={2}>
+            {category.title}
+          </Text>
+          <Text variant="intro" as="p">
+            {category.description}
+          </Text>
+        </Box>
 
-        <Hidden above="sm">
-          <Box onClick={scrollToSelectOnMobile}>
-            <Select
-              label={t.serviceCategories}
-              defaultValue={{
-                label: category.title,
-                value: category.slug,
-              }}
-              onChange={({ value }: Option) => {
-                const slug = value as string
-
-                Router.push(
-                  makePath('ArticleCategory', '/[slug]'),
-                  makePath('ArticleCategory', slug),
-                )
-              }}
-              isSearchable={false}
-              options={categoryOptions}
-              name="categories"
-            />
-          </Box>
-        </Hidden>
-        <Text variant="h1" as="h1" paddingTop={[4, 4, 0]} paddingBottom={2}>
-          {category.title}
-        </Text>
-        <Text variant="intro" as="p">
-          {category.description}
-        </Text>
-      </CategoryLayout>
+        <Box display={['block', 'block', 'none']} marginBottom={4}>
+          <Navigation
+            baseId="mobileNav"
+            colorScheme="purple"
+            isMenuDialog={true}
```

Comment:
no need adding `={true}` for booleans
```suggestion
            isMenuDialog
```